File: MultiFlex/Frontend/Multiflex.Frontend/MultiFlex.Frontend.WebApp/Models/LieferantDto.cs
using System.Collections.Generic;

namespace Multiflex.Frontend.WebApp.Models
{
    public class LieferantDto
    {
        public int id { get; set; }
        public int lieferzeit { get; set; }
        public string name { get; set; }
        public List<int> waren_ids { get; set; }
        public string weblink { get; set; }

    }
}

File: MultiFlex/API/JavaRestApi/src/main/java/org/acme/DTO/LieferantDto.java
package org.acme.DTO;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.io.Serializable;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class LieferantDto implements Serializable {
    private Integer id;

    private String name;

    private String weblink;

    private Integer lieferzeit;

    private List<Integer> waren_ids;
}
